### PR TITLE
updated RAM

### PR DESCRIPTION
**Pull request by jerae-duffin**
Updated RAM and TOC

### CODE REVIEW

File: docker-hub/registry-access-management.md [hunk 1]
```diff
@@ -39,18 +35,18 @@ To ensure that each org member uses Registry Access Management on their local ma
...
 
-   After you’ve created the registry.json file and deployed it onto the developers’ machines, you can verify whether the changes have taken effect by asking the developers to start Docker Desktop.
+## Verify the restrictions
 
-   If the configuration is successful, Docker Desktop prompts the developer to authenticate using the organization credentials on start. If the developer fails to authenticate, or authenticates as a developer in the wrong organization they will see an error message, and they will be denied access to Docker Desktop.
+   The new Registry Access Management policy should be in place after the developer successfully authenticates to Docker Desktop using their organization credentials. The developer can attempt to pull an image from a disallowed registry via the Docker CLI. They will then receive an error message that your organization disallows this container registry.
```
Comment (christophermclellan) [inline docker-hub/registry-access-management.md:49]: `that your organization has disallowed this registry.`

File: docker-hub/registry-access-management.md [hunk 2]
```diff
@@ -39,7 +35,7 @@ To ensure that each org member uses Registry Access Management on their local ma
 1. Download the latest version of Docker Desktop, and then
 2. Create a `registry.json` file.
 
-Download Docker Desktop 4.5 or a later release.
+Download Docker Desktop 4.8 or a later release.
```
[the remark this answers is not in the document]
Comment (jerae-duffin) [reply docker-hub/registry-access-management.md:38]: Resolved. Moved content above as requirement and the links are current with Windows/Mac install page that's currently on Docs

File: docker-hub/registry-access-management.md (hunk 1 above)
Comment (jerae-duffin) [reply docker-hub/registry-access-management.md:50]: Updated

**Review by usha-mandya — APPROVED** [review]
LGTM

**Pull request merged**